IntegrationTests/UI.IntegrationTests/GitUICommands/RunCommandTests.cs

```diff
@@ -363,5 +363,14 @@ private static void ClickButton(Form form, string buttonName)
 
         private static void SetText(Form form, string controlName, string value)
             => form.FindDescendantOfType<Control>(control => control.Name == controlName).Text = value;
+
+        [Test]
+        public void RunCommandBasedOnArgument_dot()
+        {
+            var selected = ObjectId.Random();
+            var first = ObjectId.Random();
+            var otherIgnored = ObjectId.Random();
```

Comment:
What is the purpose of those unused instances?

Reply:
Thought to extend the unit tests. But, since it's handled with a function, removed it.

Reply:
Removed in the latest push.